src/pages/index.astro

```diff
@@ -24,13 +23,11 @@ import TBDPlaceholder from "../../public/assets/tbd_placeholder.webp";
 				<h1>Your conversations,<br />your way.</h1>
 			</div>
 			<div class="buttons">
-				<a href="https://cloud.disroot.org/apps/forms/s/FM44AkPn9wdHgXM2DPj3oTHY"
-					><button id="signup" class="ripple"
-						>Sign up for early access <Icon
-							name="material-symbols:arrow-forward"
-						/></button
-					></a
-				>
+				<button id="signup" class="ripple" onclick="window.location.href='https://cloud.disroot.org/apps/forms/s/FM44AkPn9wdHgXM2DPj3oTHY';"
```

Comment:
I'm sorry but I have to NACK this PR due to this alone. Using JS (nonstandard) to handle something the previous HTML (standard) was perfectly able to do feels like a worse solution.